### ISSUE TITLE
No answer appears when click on the FAQ card

### ISSUE THREAD

**Issue by ydah** [r7kamura/yattecast#38]
I've accessed the following page.
https://r7kamura.github.io/yattecast/

There, I clicked on a clickable card to see the answer to a "Frequently Asked Questions" but the answer did not appear.

![](https://user-images.githubusercontent.com/13041216/168450304-cb999155-319c-48a3-aa0f-94d98cce47a9.png)

Looking at the following, the answers certainly seem to exist.

https://github.com/r7kamura/yattecast/blob/c1f4439e5f79dabd1b86a2a52e53e87735550c77/pages/index.js#L184-L199

**Comment by r7kamura** [comment]
Yes, I know this problem, but haven't debug it yet. It used to work on the old days, but it stopped working at some point.
